inc/formatconvert.class.php

```diff
@@ -2050,7 +2050,9 @@ static function networkequipmentInventoryTransformation($array) {
                                                     'SYSDESCR' => 'sysdescr',
                                                     'MODEL'    => 'model',
                                                     'IP'       => 'ip',
-                                                    'SYSMAC'   => 'mac',
+                                                    //'SYSMAC'   => 'mac',
+                                                    'SYSMAC'   => 'sysmac',
+                                                    'MAC'      => 'mac',
```

Comment:
That's the problem, when we are in CDP we don't know the mac addresses connected to the port, for this reason i made a pull request in fusioninventory-agent, check this: https://github.com/fusioninventory/fusioninventory-agent/pull/201/files